org.openhab.binding.kostalinverter/src/main/resources/ESH-INF/thing/PIKO1020.xml

```diff
@@ -0,0 +1,82 @@
+            <channel typeId="statistic-autonomy-degree-day" id="autonomyDegree_Day" ></channel>
+            <channel typeId="statistic-home-consumption-total" id="homeConsumption_Total" ></channel>
+            <channel typeId="statistic-own-consumption-total" id="ownConsumption_Total" ></channel>
+            <channel typeId="statistic-total-operating-time" id="totalOperatingTime" ></channel>
+            <channel typeId="device-local-current" id="current" ></channel>
+            <channel typeId="device-local-current-dir" id="currentDir" ></channel>
+            <channel typeId="device-local-charge-cycles" id="chargeCycles" ></channel>
+            <channel typeId="device-local-battery-temperature" id="batteryTemperature" ></channel>
+            <channel typeId="device-local-loginterval" id="loginterval" ></channel>
+            <channel typeId="device-local-s0-inpulse-cnt" id="s0InPulseCnt" ></channel>
+            <channel typeId="statistic-own-cons-rate-total" id="ownConsRate_Total" ></channel>
+            <channel typeId="statistic-autonomy-degree-total" id="autonomyDegree_Total" ></channel>
+            
+            <channel typeId="device-local-battery-voltage" id="batteryVoltage" ></channel>
+            <channel typeId="device-local-bat-state-of-charge" id="batStateOfCharge" ></channel>
+            <channel typeId="device-local-self-consumption" id="selfConsumption" ></channel>
+            
+            
+            <channel typeId="device-local-charge-time-end" id="chargetimeend" ></channel>
+            <channel typeId="device-local-battery-type" id="batteryType" ></channel>
+            <channel typeId="device-local-battery-usage-consumption" id="batteryUsageConsumption" ></channel>
+            <channel typeId="device-local-battery-usage-strategy" id="batteryUsageStrategy" ></channel>
+            <channel typeId="device-local-smart-battery-control" id="smartBatteryControl" ></channel>
+            <channel typeId="device-local-smart-battery-control-text" id="smartBatteryControl_Text" ></channel>
+            <channel typeId="device-local-battery-charge-time-from" id="batteryChargeTimeFrom" ></channel>
+            <channel typeId="device-local-battery-charge-time-to" id="batteryChargeTimeTo" ></channel>
+            <channel typeId="device-local-max-depth-of-discharge" id="maxDepthOfDischarge" ></channel>
+            <channel typeId="device-local-shadow-management" id="shadowManagement" ></channel>
+            <channel typeId="device-local-external-module-control" id="externalModuleControl" ></channel>
+            <channel typeId="device-local-inverter-name" id="inverterName" ></channel>
+            </channels>
+            <properties>
+            <property name="Vendor">KOSTAL Solar Electric GmbH</property>
+        </properties>
+        <config-description-ref uri="thing-type:kostalpikonewgeneration_config" />
+   </thing-type>
+</thing:thing-descriptions>
```

Comment:
Some of our build tools don't like EOF without a newline. So add a newline here.